src/hotspot/share/gc/shenandoah/shenandoahControlThread.cpp

```diff
@@ -438,8 +439,8 @@ void ShenandoahControlThread::process_phase_timings(const ShenandoahHeap* heap)
       ResourceMark rm;
       LogStream ls(lt);
       heap->phase_timings()->print_cycle_on(&ls);
-      ShenandoahEvacuationTracker::print_evacuations_on(&ls, &evac_stats.workers,
-                                                        &evac_stats.mutators);
+      evac_tracker->print_evacuations_on(&ls, &evac_stats.workers,
```

Comment:
Does this need to be per instance.

Reply:
It's a bit awkward to keep it static, and the cost of calling the instance method no worse. (We actually just make use of a single instance field `_generational` in the method. We could make that field static and leave the method static, but I am not sure it's worth doing.